jobspec/parse_task.go

```diff
@@ -12,6 +12,48 @@ import (
 	"github.com/mitchellh/mapstructure"
 )
 
+var (
+	normalTaskKeys = []string{
```

Comment:
So that the shared keys don’t fall out of sync maybe we should have a `coreTaskKeys` and have these two something like 
`normalTaskKeys = append(coreTaskKeys, ...)`

Reply:
Good idea; done!